### PR TITLE
feat(select): describe select component using design tokens

### PR DESCRIPTION
**Pull request by mjaniczko-sage**
### Proposed behaviour
Describes the select using design tokens. Removes all possible 'theme' in css files

### Current behaviour

The select use the theme styled-component property.

### Checklist

- [x] Commits follow our style guide
- [ ] Related issues linked in commit messages if required
- [ ] Screenshots are included in the PR if useful
- [x] All themes are supported if required
- [x] Unit tests added or updated if required
- [ ] Cypress automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Translations added or updated (including creating or amending translation keys table in storybook) if required
- [ ] Typescript `d.ts` file added or updated if required

#### QA

- [ ] Tested in CodeSandbox/storybook
- [ ] Add new Cypress test coverage if required
- [ ] Carbon implementation matches Design System/designs
- [ ] UI Tests GitHub check reviewed if required

### Testing instructions

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.

### CODE REVIEW

File: src/components/select/list-action-button/list-action-button.style.js [hunk 1]
```diff
@@ -1,41 +1,34 @@
...
+    width: 100%;
 
-      :hover {
-        background-color: ${theme.select.selected};
-
-        ${StyledIcon} {
-          color: ${theme.text.color};
-        }
+    :hover {
+      background-color: ${tint("var(--colorsActionMajor500")(95)};
```
Comment (jhajduk-sage) [inline src/components/select/list-action-button/list-action-button.style.js:26]: You crazy man! :D

**Review by jhajduk-sage — APPROVED** [review]

**Review by grabkowski — COMMENTED** [review]
I think that the commit message and the PR title should be:
feat(select): describe Select component using Design Tokens

**Review by grabkowski — DISMISSED** [review]

**Review by DipperTheDan — DISMISSED** [review]

**Comment by harpalsingh** [comment]
The color when using arrow keys has a different background to the hover and this should match.

Screenshot 2022-01-05 at 08 55 36

Screenshot 2022-01-05 at 08 56 27

Checked with Claire, its Utility Major 200 is the color for both, so the darker of the 2.

**Review by DipperTheDan — DISMISSED** [review]

**Review by grabkowski — APPROVED** [review]

**Review by DlgSHi — APPROVED** [review]

**Review by DipperTheDan — APPROVED** [review]

**Pull request merged**